src/test-helpers/e2e-helpers/mobile/clickThought.ts

```diff
@@ -0,0 +1,14 @@
+import getEditable from './getEditable'
+import { Browser } from 'webdriverio'
+
+/** Click the thought for the given thought value. */
+const clickThought = async (browser: Browser<'async'>, value: string) => {
+
+  const editableNode = await getEditable(browser, value)
+
+  if (!editableNode) throw new Error('editable node for the given value not found.')
```

Comment:
You could add `value` to the error message to make it a bit more useful in the test output.